front-end/public/secrets.svg

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" ?><svg data-name="Layer 1" id="Layer_1" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg"><defs><style>.cls-1,.cls-2{fill:none;}.cls-2{stroke:#000;stroke-miterlimit:10;stroke-width:0.463px;}</style></defs><title/><ellipse cx="6" cy="5.5" rx="1" ry="1.5"/><ellipse cx="10" cy="5.5" rx="1" ry="1.5"/><rect height="1.998" width="1" x="3" y="9"/><rect height="1.998" width="1" x="5" y="9"/><rect height="1.998" width="1" x="7" y="9"/><path class="cls-1" d="M10.1,12.952l-.256-2.421a.622.622,0,0,1,.618-.687l.066,0,2.421.256a.626.626,0,0,1,.375.178l.9.9a7.021,7.021,0,1,0-3.045,3.045l-.9-.9A.622.622,0,0,1,10.1,12.952Z"/><path d="M14.968,11.923a8.01,8.01,0,1,0-3.045,3.045l.849.85a.623.623,0,0,0,.88,0l2.166-2.166a.622.622,0,0,0,0-.88ZM8,15a7.021,7.021,0,1,1,6.227-3.818l-.9-.9a.626.626,0,0,0-.375-.178l-.973-.1H12V9H11v.9l-.469-.05-.066,0a.616.616,0,0,0-.465.213V9H9v2h.9l.207,1.954a.622.622,0,0,0,.179.375l.9.9A6.944,6.944,0,0,1,8,15Zm5.212-.156-2.129-2.13-.192-1.823,1.824.192,2.129,2.129Z"/><circle class="cls-2" cx="13.165" cy="13.165" r="0.5"/></svg>
```

Comment:
Newline

Reply:
this is just a little favicon

Reply:
Yea it may seem trivial, but I would add newlines to every file in general. In the industry it's usually some set rule in .prettier or setup in the linter. This guy has some good references on why newlines are important

> https://github.com/mermaid-js/mermaid/issues/400#issuecomment-254171909

Reply:
ok, thank you